apps/ai-content-generator/src/configs/launch-darkly/ldConfig.ts

```diff
@@ -0,0 +1,16 @@
+const ldConfig = {
+  clientSideID: `${import.meta.env.VITE_LAUNCHDARKLY_CLIENT_ID}`,
+  deferInitialization: false,
+  flags: {
+    'integrations-ai-content-generator-v-2': true,
```

Comment:
Do we have a naming convention for these? I lost my launch darkly access so i actually can't check. 🤷